### PR TITLE
Division management #29724566985606

### PR DESCRIPTION
**Pull request by HoangThiTuanDung**

### CODE REVIEW

File: app/Controller/DivisionsController.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+				}
+			} else {
+				$this->Session->setFlash(__('Your data is NOT available.'));
+			}
+		}
+		$this->loadModel('User');
+		$this->loadModel('Position');
+		$position = $this->Position->getByName('Manager');
+		if (!$position) {
+			$this->Session->setFlash('Please create Manager Position');
```
Comment (tranluan91) [inline app/Controller/DivisionsController.php]: Please move flash message into ```__()```

File: app/Model/Position.php [hunk 2]
```diff
@@ -21,4 +21,8 @@ class Position extends AppModel {
 			]
 		],
 	];
+
+	public function getByName($name) {
```
Comment (tranluan91) [inline app/Model/Position.php]: Why did you create this function? Use ```findByName()``` instead it

**Comment by tranluan91** [comment]
OK

**Comment by dinhhoanglong91** [comment]
OK

**Pull request merged**